### PR TITLE
Cleaning up AntiForgery for ShoppingCarter

### PR DESCRIPTION
**Pull request by ianhong**
@Praburaj, @Eilon, @harshgMSFT, @GrabYourPitchforks 

I updated ShoppingCarter as we concluded. Please review this one.

PS. In addition to what we have discussed, I updated the controller to use the property injection for AntiForgery. 

Thanks,
Ian

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@ianhong__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Youngjune Hong). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: src/MusicStore/Controllers/ShoppingCartController.cs [hunk 1]
```diff
@@ -55,24 +57,21 @@ public class ShoppingCartController : Controller
...
                 if (tokens != null && tokens.Length == 2)
                 {
                     cookieToken = tokens[0];
                     formToken = tokens[1];
                 }
             }
-
-            var antiForgery = Context.RequestServices.GetService<AntiForgery>();
-            antiForgery.Validate(Context, new AntiForgeryTokenSet(formToken, cookieToken));
+            AntiForgery.Validate(Context, new AntiForgeryTokenSet(formToken, cookieToken));
```
Comment (Praburaj) [inline src/MusicStore/Controllers/ShoppingCartController.cs]: Put a new line here.

**Comment by Praburaj** [comment]
Change looks good to me. If @Eilon and @harshgMSFT are happy you can push.

**Comment by harshgMSFT** [comment]
:shipit:

**Comment by ianhong** [comment]
29909e15f33b22db1fe638457315199e0a46eeea

File: src/MusicStore/Controllers/ShoppingCartController.cs [hunk 2]
```diff
@@ -55,15 +57,14 @@ public class ShoppingCartController : Controller
         [HttpPost]
         public async Task<IActionResult> RemoveFromCart(int id, CancellationToken requestAborted)
         {
-            var formParameters = await Context.Request.ReadFormAsync();
-            var requestVerification = formParameters["RequestVerificationToken"];
-            string cookieToken = null;
-            string formToken = null;
+            var cookieToken = string.Empty;
+            string formToken = string.Empty;
```
Comment (Eilon) [inline src/MusicStore/Controllers/ShoppingCartController.cs]: Can also `var` this.

Comment (ianhong) [inline src/MusicStore/Controllers/ShoppingCartController.cs]: I included this in https://github.com/aspnet/MusicStore/commit/1b502cbee7b86709a0b9a0afd4de4f3c97db52ae

Thanls

File: .vs/config/applicationhost.config [hunk 3]
```diff
@@ -0,0 +1,1023 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Praburaj) [inline .vs/config/applicationhost.config]: @ianhong do you need this?

Comment (ianhong) [inline .vs/config/applicationhost.config]: Oh...it's been checked in yesterday. and I didn't include this file. :-) Thanks!

**Pull request closed without merging**